tuning/sft_trainer.py

```diff
@@ -254,6 +260,30 @@ def train(
         )
         callbacks.append(tc_callback)
 
+    if train_args.packing:
```

Comment:
why is this code duplicated ? its already in line 215 above?

Reply:
It was a merging issue. Fixed.